src/credentials/FabricTable.h

```diff
@@ -385,6 +385,11 @@ class DLL_EXPORT FabricTable
          **/
         virtual void OnFabricCommitted(const FabricTable & fabricTable, FabricIndex fabricIndex){};
 
+        /**
+         * Gets called when a fabric is newly added to Fabric Table.
```

Comment:
This needs to document the exact behavior, which is that OnFabricUpdated will _also_ get called in that case.  And OnFabricUpdated documentation should probably be modifiied to make it clear that "changed" includes "added".